plugins/modules/mysql_user.py

```diff
@@ -74,6 +74,12 @@
       - Whether binary logging should be enabled or disabled for the connection.
     type: bool
     default: yes
+  db_context:
```

Comment:
not completely convinced by the name, as it doesn't seem clear at first glance, but can't find something better right now 🤔

Reply:
Yeah, me neither... as usual, one of the 2 hardest things in IT... i had login_db in my first draft but i wasn´t convinced. I´m up for suggestions.

Reply:
`force_context` ?

Reply:
`db_context` works for me, though I'd be happy with any name you'll agree on:)